tools/metashrimps/DataPaper.xml

```diff
@@ -0,0 +1,83 @@
+<tool id="metashrimp_datapaper" name="Production of a draft of Data Paper" version="0.1.0+galaxy0" profile="22.05">
+    <description>Create a draft of Data Paper from metadata</description>
+    <requirements>
+	   <requirement type="package">r-base</requirement>
```

Comment:
Don't remove version here.

What you can do is ... outside of Galaxy do:

```bash
conda create -n foo phantomjs pandoc gmp r-mapview .....
```
Then you get an ouput and you take the versions numbers from that output and add it here.